Docs/ContinuousIntegration.md

```diff
@@ -22,9 +22,9 @@ Normally when using a code formatter like CSharpier, you'll want to ensure that
 2. Use your preferred CI/CD tool to run the following commands.
    ```bash
    dotnet tool restore
-   dotnet-csharpier --check
+   dotnet tool run dotnet-csharpier --check .
    ```   
-   An example of a github action to accomplish this
+   An example of a GitHub Actions to accomplish this
```

Comment:
This sounds a little weird, and the original version wasn't quite right either.
After looking at the doc for Github Actions, maybe it should be something like
```
An example Github Actions workflow to accomplish this

Reply:
Absolutely! I'm not a native English speaker, so I lacked a perspective on the naturalness of the text.